tools/nd_freeze

```diff
@@ -314,8 +314,9 @@ sub write_snapshot_sources {
         my $line_prefix = "";
         $line_prefix = "#" if ($sources{$key}{Label} eq 'Debian-Security');
 
-        print $fp "${line_prefix}deb http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}\n";
-        logg("debug", "Adding 'deb http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}' to $snapshots_sources_file");
+        my $trusted_option = ( is_minimum_apt_version("0.9.40") ? "[trusted=yes]" : "" );
+        print $fp "${line_prefix}deb $trusted_option http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}\n";
+        logg("debug", "Adding 'deb $trusted_option http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}' to $snapshots_sources_file");
```

Comment:
Could you please assign the line content to a variable and then use it for print and debug - duplication is evil! ;-)